Source/santa-driver/SantaDecisionManager.cc

```diff
@@ -427,6 +445,89 @@ void SantaDecisionManager::DecrementListenerInvocations() {
   OSDecrementAtomic(&listener_invocations_);
 }
 
+# pragma mark Monitoring PIDs
+
+void SantaDecisionManager::ForgetCompilerPid(pid_t pid) {
+  compiler_pid_set_->remove(pid);
+}
+
+// Arguments that we pass to pid_monitor_thread.
+typedef struct {
+  pid_t pid;                  // process to monitor
+  struct timespec ts;         // time to sleep between checks
+  SantaDecisionManager *sdm;  // reference to SantaDecisionManager
+} pid_monitor_info;
+
```

Comment:
Fixed.  (BTW, this struct definition and the following function might should be moved somewhere else since they're not part of the class, I just liked having all the process monitoring code close together).